disperser/controller/dispatcher_metrics.go

```diff
@@ -181,6 +191,31 @@ func newDispatcherMetrics(registry *prometheus.Registry) *dispatcherMetrics {
 		"send_to_validator",
 		false)
 
+	// Verify that thresholds are sane
+	for _, threshold := range importantSigningThresholds {
+		if threshold < 0 || threshold > 1 {
+			return nil, fmt.Errorf("threshold %f is not between 0.0 and 1.0", threshold)
+		}
+	}
+	sort.Float64s(importantSigningThresholds)
+
+	// Add thresholds for 0.0 and 1.0, if missing.
+	if len(importantSigningThresholds) == 0 || importantSigningThresholds[0] != 0.0 {
+		importantSigningThresholds = append([]float64{0.0}, importantSigningThresholds...)
+	}
+	if importantSigningThresholds[0] != 1.0 {
+		importantSigningThresholds = append(importantSigningThresholds, 1.0)
+	}
+
+	signatureThresholds := promauto.With(registry).NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: dispatcherNamespace,
+			Name:      "signature_thresholds",
```

Comment:
`signature_thresholds` isn't super clear about what this does. What do you think about `blob_count_reaching_thresholds` as an alternative name, or something similar?